### PR TITLE
feature(locksmith, unlock-app): Multiple checkins

### PR DESCRIPTION
**Pull request by julien51**
# Description

Adding suppprt for multiple checkins in the verifier.

# Issues

Fixes #
Refs #https://github.com/unlock-protocol/unlock/issues/12992

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: unlock-app/src/components/interface/locks/Manage/elements/MetadataCard.tsx [hunk 1]
```diff
@@ -283,10 +283,24 @@ export const MetadataCard = ({
...
+    if (checkInTimestamp) {
+      // checked in from this UI
+      return checkInTimestamp
+    }
+    // Check the metadata
     const [_, checkInTimeValue] =
       Object.entries(metadata)?.find(([key]) => key === 'checkedInAt') ?? []
-    if (checkInTimestamp) return checkInTimestamp
-    if (!checkInTimeValue) return null
+    console.log(checkInTimeValue)
```
Comment (searchableguy) [inline unlock-app/src/components/interface/locks/Manage/elements/MetadataCard.tsx:293]: remove?

File: locksmith/src/controllers/v2/ticketsController.tsx [hunk 2]
```diff
@@ -57,15 +57,16 @@ export class TicketsController {
...
-          error: 'Ticket already checked in',
-        })
+      const checkedInAt = []
+      if (!data?.metadata?.checkedInAt) {
+        checkedInAt.push(new Date().getTime())
+      } else if (typeof data?.metadata?.checkedInAt === 'number') {
+        checkedInAt.push(data?.metadata?.checkedInAt, new Date().getTime())
+      } else if (Array.isArray(data?.metadata?.checkedInAt)) {
+        checkedInAt.push(...data.metadata.checkedInAt, new Date().getTime())
       }
```
Comment (searchableguy) [inline locksmith/src/controllers/v2/ticketsController.tsx:70]: can you add some tests for this part specifically in the endpoint?

**Review by searchableguy — APPROVED** [review]
LGTM but one addition

File: unlock-app/src/components/interface/locks/Manage/elements/MetadataCard.tsx (hunk 1 above)
Comment (julien51) [inline unlock-app/src/components/interface/locks/Manage/elements/MetadataCard.tsx:293]:
```suggestion
```

File: locksmith/src/controllers/v2/ticketsController.tsx (hunk 2 above)
Comment (julien51) [reply locksmith/src/controllers/v2/ticketsController.tsx:70]: Ha good point! Let me add that a bit later as I need to merge this ASAP

**Pull request merged**